### PR TITLE
[Review Needed]Pytest Stylecore/test_topology.py

### PR DESCRIPTION
**Pull request by utkbansal**
Fixes #

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: testsuite/MDAnalysisTests/core/test_topology.py [hunk 1]
```diff
@@ -29,17 +26,15 @@
...
 
 
-class TestTransTable(TestCase):
-    def setUp(self):
-        # Reference data
-        self.Ridx = np.array([0, 0, 2, 2, 1, 1, 3, 3, 1, 2])
-        self.Sidx = np.array([0, 1, 1, 0])
-        self.tt = TransTable(10, 4, 2, self.Ridx, self.Sidx)
+class TestTransTable(object):
+    Ridx = np.array([0, 0, 2, 2, 1, 1, 3, 3, 1, 2])
```
Comment (richardjgowers) [inline testsuite/MDAnalysisTests/core/test_topology.py]: these aren't used anywhere but the fixture, so can move these into there

File: testsuite/MDAnalysisTests/core/test_topology.py [hunk 2]
```diff
@@ -142,8 +136,8 @@ def test_move_atom_simple(self):
         assert_equal(len(tt.residues2atoms_1d(0)), 1)  # 1 fewer here
         assert_equal(len(tt.residues2atoms_1d(3)), 3)  # 1 more here
 
-    def test_move_residue_simple(self):
-        tt = self.tt
+    def test_move_residue_simple(self, tt):
+        tt = tt
```
Comment (richardjgowers) [inline testsuite/MDAnalysisTests/core/test_topology.py]: I'm not sure this line is necessary

Comment (orbeckst) [inline testsuite/MDAnalysisTests/core/test_topology.py]: Is this English politeness saying "It's not necessary, remove it"?

File: testsuite/MDAnalysisTests/core/test_foo.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import pytest
```
Comment (richardjgowers) [inline testsuite/MDAnalysisTests/core/test_foo.py]: I guess this isn't meant to be here

**Pull request merged**